README.md

```diff
@@ -2,3 +2,6 @@
 This project serves as a basic guide for working with  GitHub. It is based on the Hello World GitHub tutorial   https://guides.github.com/activities/hello-world/
 
 Now, let us update this file by adding some notes. Again , the purpose is to understand how the whole think works
+
+
+THIS IS THE SECOND TIME CREATING A BRANCH
```

Comment:
A new comment line

Reply:
Maybe I should merge this ...

Reply:
...or maybe not

Reply:
why?

Reply:
...why not?

Reply:
...sounds a little schitzofrenic talking to yourself just to learn a bloody tool, right?